setup.py

```diff
@@ -9,7 +9,7 @@
 
 setuptools.setup(
     name="tf-fastmri-data",
-    version="0.0.3",
+    version="0.0.5",
```

Comment:
Why the big jump?

Reply:
I think current master is at 0.0.4 right? Did I miss something?

Reply:
Oops, I had it at 0.0.4 already, i messed up and did another bump. Will fix that